db/seeds.rb

```diff
@@ -0,0 +1,86 @@
+require 'csv'
+
+# Jeremy's example from Record-Sandbox
+# FILE_PATH = 'db/seed_csvs/markets.csv'
+#
+# CSV.foreach(FILE_PATH, headers: true) do |row|
+#  row_hash = row.to_h
+#  # artist_name = row_hash.delete("artist")
+#  # artist = Artist.find_or_create_by(name: artist_name)
+#  artist.albums << Album.create(row_hash)
+# end
```

Comment:
You could also follow the Jeremy example above and just convert the rows to a hash and add the items to your database as a hash. The way you did also works but requires a lot more typing! Be a lazier programmer! 😄